wp-content/themes/wp-starter/vite.config.js

```diff
@@ -3,9 +3,11 @@ import path from 'path';
 import liveReload from 'vite-plugin-live-reload';
 import generateThemeJSON, { buildJSON } from './src/theme-json/generate_theme.js';
 
+const THEME = '/wp-content/themes/wp-starter';
+
 export default defineConfig(({ command }) => ({
 	root: 'src',
-	base: command === 'serve' ? '' : '/dist/',
+	base: command === 'serve' ? '' : THEME + '/dist/',
```

Comment:
@bd-viget @empee3 I did not have the theme directory in the build so it was looking for it at the root.